pkg/controllers/vdb/restart_reconcile.go

```diff
@@ -45,6 +47,8 @@ const (
 	AdminToolsMapFile = "/opt/vertica/config/ipMap.txt"
 	// Constant for an up node, this is taken from the STATE colume in NODES table
 	StateUp = "UP"
+	// Percent of livenessProbe time to wait when requeuing due to waiting on livenessProbe
+	PctOfLivenessProbeWait = 0.25
```

Comment:
Is this a random value?

Reply:
Yeah, pretty much. It's just a rough heuristic of when to retry the reconcile iteration. I'll update the comment.